tools/ak2-core.sh

```diff
@@ -107,6 +107,7 @@ unpack_ramdisk() {
     ui_print " "; ui_print "Unpacking ramdisk failed. Aborting..."; exit 1;
   fi;
   test ! -z "$(ls /tmp/anykernel/rdtmp)" && cp -af /tmp/anykernel/rdtmp/* $ramdisk;
+  rm -f $ramdisk/placeholder;                           
```

Comment:
This should probably just be done as part of a clean up commit in the user's fork, not much point otherwise.